handlers/soft-opt-in-consent-setter/src/main/scala/com/gu/soft_opt_in_consent_setter/HandlerIAP.scala

```diff
@@ -5,7 +5,7 @@ import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
 import com.gu.soft_opt_in_consent_setter.models._
 import com.typesafe.scalalogging.LazyLogging
 import io.circe.ParsingFailure
-import io.circe.{Decoder, DecodingFailure}
+import io.circe.{Decoder}
```

Comment:
total nitpick, I'm surprised scalafmt didn't complain
```suggestion
import io.circe.Decoder
```